packages/cli/src/databases/repositories/folder.repository.ts

```diff
@@ -226,4 +226,15 @@ export class FolderRepository extends Repository<FolderWithWorkflowsCount> {
 			query.skip(options.skip ?? 0).take(options.take);
 		}
 	}
+
+	async findOneOrFailFolderInProject(folderId: string, projectId: string): Promise<Folder | null> {
```

Comment:
I think typings are wrong here, `findOneOrFail` rejects if it doesn't find any entities so this shouldn't ever return null?

Also the typings of this repository seem a bit confusing overall, the repository class is typed to extend `Repository<FolderWithWorkflowCount>` which makes the `this.findOneOrFail` also return `FolderWithWorkflowCount`, but I guess we're not loading the workflows count here for instance as we're just typing this to return only `Folder`? This doesn't perhaps need to change within this PR, but it just feels like we're doing something wrong here.